services/web-app/pages/contribute/components.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright IBM Corp. 2021, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { NextSeo } from 'next-seo'
+
+import RemoteMdxLoader from '@/components/remote-mdx-loader'
+import { getRemoteMdxData } from '@/lib/github'
+
+const RemoteMdxPage = ({ source }) => {
+  const seo = {
+    title: 'Components'
+  }
+
+  return (
+    <>
+      <NextSeo {...seo} />
+      <RemoteMdxLoader source={source} />
+    </>
+  )
+}
+
+export const getStaticProps = async () => {
+  const mdxSource = await getRemoteMdxData(
+    {
+      host: 'github.com',
+      org: 'carbon-design-system',
+      repo: 'carbon-website',
+      library: 'carbon-website',
+      ref: 'main'
+    },
+    '/src/pages/contributing/component/index.mdx'
```

Comment:
the links to components here are pointing to /[asset]/usage which 404s in platform, we don't have that tab yet, are we cool with that? 🤔

Reply:
Checked in with Aubrey, we're good with 404s in remote pages for now 👍